dev/reference/configuration/tikv-server/configuration-file.md

```diff
@@ -324,6 +324,16 @@ Configuration items related to Raftstore
 + Default value: `3s`
 + Minimum value: `0`
 
+### `raftstore.hibernate-regions` (**Experimental**)
+
++ Enables or disables Hibernate Region. When enabled, a Region idle for a long time is automatically set as hibernated. This reduces the extra overhead caused by heartbeat messages between the Raft leader and the followers for idle Regions. You can use `raftstore.peer-stale-state-check-interval` to modify the the heartbeat interval between the leader and followers of hibernated Regions.
```

Comment:
Since "hibernate" is an intransitive verb, "hibernated" might be inappropriate.

Reply:
I think we can use it here - to indicate a status, not in the sense of original verb meaning. But I agree it might not be the best solution - the feature name is not a correct usage, either.

Reply:
OK.